README.md

```diff
@@ -12,12 +12,14 @@ Each workflow has an associated documentation file that explains its purpose, us
 
 Here are the available workflows and their associated documentation:
 
-- [Reusable Docker Container Build](.github/workflows/workflow-build-container.md)
-- [Reusable Docker Container Deployment to Google Cloud Run](.github/workflows/workflow-deployment-gcr.md)
+- [Reusable Docker Container
+  Build](.github/workflows/workflow-build-container.md)
```

Comment:
not sure why the code block for the link wraps here?

Reply:
done